acos_client/v30/slb/virtual_port.py

```diff
@@ -109,8 +109,10 @@ def _set(
             params['port']['template-virtual-port'] = virtual_port_templates.get('template-virtual-port', None)
             if protocol in ['http', 'https']:
                 params['port']['template-http'] = virtual_port_templates.get('template-http', None)
-            else:
+            elif protocol in ['tcp','TCP']:
```

Comment:
Please run pep8 for this file. May give `E231`

Reply:
Fixed in new commit. FYI pep8 has been replaced by pycodestyle.